src/build.py

```diff
@@ -1233,15 +1241,15 @@ def Summary(repos):
       buildbot.Link('lkgr.json', cloud.Upload(info_file, lkgr_file))
 
 
-def AllBuilds(use_asm=False):
+def AllBuilds(use_asm=False, run_tests=False):
```

Comment:
No reason really, just following the existing convention.  We can change it later if the number of args gets out of the control.